### PR TITLE
Set new Tezos RPC nodes

### PR DESCRIPTION
**Pull request by asiia-trilitech**
## Proposed changes

Issues were found related to the RPX nodes we ='re using at the moment. We're switching to new nodes controlled by ECAD labs.

[Security review task link #48](https://github.com/trilitech/umami-security-review/issues/48)
[Security review task link #47](https://github.com/trilitech/umami-security-review/issues/47)

## Types of changes

- [x] Bugfix
- [ ] New feature
- [ ] Refactor
- [ ] Breaking change
- [ ] UI fix

### CODE REVIEW

File: apps/embed-iframe-mainnet/vercel.json
Comment (asiia-trilitech) [inline apps/embed-iframe-mainnet/vercel.json:1]: Was it just changed to  https://mainnet.tezos.ecadinfra.com & https://ghostnet.tezos.ecadinfra.com/?

https://taquito.io/docs/rpc_nodes/

File: apps/web/src/components/MasterPasswordModal/MasterPasswordModal.test.tsx [hunk 1]
```diff
@@ -9,9 +9,16 @@ describe("<MasterPasswordModal />", () => {
     await renderInModal(<MasterPasswordModal onSubmit={mockOnSubmit} />);
 
     await user.type(screen.getByLabelText("Password"), "testpassword");
-    await user.click(screen.getByRole("button", { name: "Submit" }));
+    await waitFor(
+      async () => {
+        await user.click(screen.getByRole("button", { name: "Submit" }));
+      },
+      {
+        timeout: 5000,
```
Comment (asiia-trilitech) [inline apps/web/src/components/MasterPasswordModal/MasterPasswordModal.test.tsx:17]: Why do we need timeout here?

Comment (asiia-trilitech) [reply apps/web/src/components/MasterPasswordModal/MasterPasswordModal.test.tsx:17]: Have you tried act like in the other tests?

File: packages/state/src/migrations.ts
Comment (asiia-trilitech) [inline packages/state/src/migrations.ts:1]: Please add a test for the migration

File: apps/web/src/setupTests.ts
Comment (asiia-trilitech) [inline apps/web/src/setupTests.ts:1]: nit: why test setup was changed?

**Review by asiia-trilitech — COMMENTED** [review]
Thank you for the change @OKendigelyan!

I think the node urls was just updated to https://mainnet.tezos.ecadinfra.com/ & https://ghostnet.tezos.ecadinfra.com/ - might be for the best to just go with the new ones to avoid changes in the future 
https://taquito.io/docs/rpc_nodes/

And please check the comments for some questions 🙁 

Overall code looks good - let me check how embed is working and get back to you

File: apps/web/src/setupTests.ts
Comment (OKendigelyan) [reply apps/web/src/setupTests.ts:1]: test changes fix flaky tests timeout issues and some other issues related to axios

File: apps/embed-iframe-mainnet/vercel.json
Comment (ajinkyaraj-23) [reply apps/embed-iframe-mainnet/vercel.json:1]: yes

**Review by ajinkyaraj-23 — COMMENTED** [review]
change the ecad rpc url to latest format.

**Review by ajinkyaraj-23 — APPROVED** [review]
LGTM.

**Pull request merged**